docs/hooks/use-content-editable-false.md

```diff
@@ -76,8 +76,9 @@ const {insertContentEditableFalseElement} = useContentEditableFalse({processCont
 
 ### Arguments
 `useContentEditableFalse` can be called with no arguments or with an object with the following properties:
-- `processContentEditableElement`: A function that will be called with an unmodifiable dom element every time that one is inserted.
-  This is used to style or change the element however you'd like to. Defaults to no-op.
+- `processContentEditableElement`: A function that will be called with each unmodifiable dom element in the rich text editor. This can be used
```

Comment:
The new wording clarifies that the processElement functions get called for initialHTML and setHTML() html, in addition to being called when an element is inserted into the DOM via hook.